### ISSUE TITLE
Timecard single day entry gives Bad Date error...

### ISSUE THREAD

**Issue by JigmeDatse** [ledgersmb/LedgerSMB#2656]
When entering a new timecard on master I get the error:

```
Bad date (2017-03-01 00:00:00-08) at /home/jrasku/src/ledgersmb/LedgerSMB/tools/../lib/LedgerSMB/PGDate.pm line 270.
```

My post data is:

```
action:timecard_screen
bu_class_id:2
num_days:1
date_from:2017-03-01
```

I have the user, and server using Canada/English set for the user, and dataset...

**Comment by JigmeDatse** [comment]
Browser is en-UK.

**Comment by JigmeDatse** [comment]
This shows up when I try to display a timecard as well.

**Comment by sbts** [comment]
@ylavoie have we missed a commit that fixed this in 1.5?
@JigmeDatse can you please test in 1.5 with the same config

**Comment by JigmeDatse** [comment]
I see the same issue with pulling 1.5 in.

**Comment by JigmeDatse** [comment]
I think that @ylavoie mentioned that I was the first person to find this, not sure what it is related to.  I'm not sure that I have done anything with it since I first found it, and he had a direction he was heading with it.

**Comment by JigmeDatse** [comment]
I noticed that the timecard gets posted (my list is getting long).  It gives the error.  I don't know if the reason it gives the error is because the timecard is getting displayed after having posted the timecard?  It's my suspicion that is what is happening, and maybe looking to see anything with the posting of the timecard would be counterproductive.

**Comment by ehuelsmann** [comment]
Fixed in 1.8 through 2212d3456fc229c741d5bbadef89f455c159230e
